Pac-Man-2020/Assets/Scripts/GhostController.cs

```diff
@@ -294,7 +307,7 @@ private void releaseGhosts()
 
                 //now that we have the right PacFourAheadPosition we want, we can find its distance to the ghost...
 
-                float tempDistance = (PacNAheadPosition - nodePos).sqrMagnitude; //distance from n pills ahead of pacman to the node we are currently iterating over
+                float tempDistance = Mathf.Abs((PacNAheadPosition - nodePos).sqrMagnitude); //distance from n pills ahead of pacman to the node we are currently iterating over
```

Comment:
Heads up, Square Magnitude is already always positive

Reply:
true I'll change that